src/components/Button.js

```diff
@@ -2,7 +2,7 @@ import React from 'react'
 
 const Button = (props) => {
     return (
-        <button >
+        <button className={!props.special ? "button" : "long-button"} style={{background: props.color}}>
```

Comment:
Why not:
```javascript
props.special ? 'long-button' : 'button'
```

Also, we are using sass aren't we? If so then we should not set the style of `Button` using the `style` attribute. Rather we should have a scss module for this.